profiles/arch/base/package.use.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Lars Wendler <polynomial-c@gentoo.org> (2021-07-09)
+# dev-libs/libbpf lacks keywords for most arches
```

Comment:
It's keyworded in most places now, just now alpha. Could we just mask in certain arches accordingly because they're in the minority?